Explorer/Assets/DCL/Minimap/MinimapController.cs

```diff
@@ -56,10 +57,23 @@ protected override void OnViewInstantiated()
         {
             viewInstance.expandMinimapButton.onClick.AddListener(ExpandMinimap);
             viewInstance.minimapRendererButton.onClick.AddListener(() => mvcManager.ShowAsync(ExplorePanelController.IssueCommand(new ExplorePanelParameter(ExploreSections.Navmap))).Forget());
+            viewInstance.sideMenuButton.onClick.AddListener(OpenSideMenu);
+            viewInstance.sideMenu.SetActive(false);
         }
 
-        private void ExpandMinimap() =>
-            viewInstance.minimapContainer.gameObject.SetActive(!viewInstance.minimapContainer.gameObject.activeSelf);
+        private void ExpandMinimap()
+        {
+            GameObject gameObject;
+            (gameObject = viewInstance.minimapContainer.gameObject).SetActive(!viewInstance.minimapContainer.gameObject.activeSelf);
+            viewInstance.arrowDown.SetActive(!gameObject.activeSelf);
+            viewInstance.arrowUp.SetActive(gameObject.activeSelf);
+            sideMenuController = new SideMenuController(viewInstance.sideMenuView);
+        }
+
+        private void OpenSideMenu()
+        {
+            viewInstance.sideMenu.SetActive(!viewInstance.sideMenu.activeSelf);

```

Comment:
Instead of being accessing continously to `viewInstance.sideMenu...`, maybe it would be clearer to implement little functions in the view to be accessed from the controller: `SetSideMenuActive(bool)`, `ToggleSideMenu()`, etc.

Reply:
we decided that in the new MVC way views do not hold any logic, it's all handled by the controller